.github/workflows/nightly-build-publish.yml

```diff
@@ -14,34 +14,124 @@ on:
     branches: 
       - master
 
+env:
+  IMAGE: quay.io/eclipse/che-devfile-registry
+  VERSION: nightly
+
 jobs:
   nightly-build-publish:
     runs-on: ubuntu-latest
+    continue-on-error: true
+    strategy:
+      fail-fast: false
+      matrix:
+        arch: [amd64,arm64,ppc64le,s390x]
+    outputs:
+      amd64: ${{ steps.result.outputs.amd64 }}
+      arm64: ${{ steps.result.outputs.arm64 }}
+      ppc64le: ${{ steps.result.outputs.ppc64le }}
+      s390x: ${{ steps.result.outputs.s390x }}
     steps:
-      - 
-        name: Clone source code
+      -
+        name: "Checkout source code"
         uses: actions/checkout@v1
         with:
-          fetch-depth: 1
```

Comment:
Any specific reason to change this?

Reply:
No, kept release and nightly fetch-depth inline but I guess for nightly job we can have fetch-depth as 1 such that only one commit is fetched.